src/model/paths_types.ml

```diff
@@ -42,7 +42,8 @@ module Identifier = struct
 
   type source_location_pv =
     [ `SourceLocationMod of source_page
-    | `SourceLocation of source_page * DefName.t ]
+    | `SourceLocation of source_page * DefName.t
+    | `SourceLocationInt of source_page * LocalName.t ]
```

Comment:
The purpose of the new constructor is not obvious, a comment would be nice. Or perhaps to rename it `SourceLocationInternal` ?

Reply:
I like the explicit name: it makes it easier to understand also in occurrences of the constructor.